css/src/mixins/media-queries.scss

```diff
@@ -40,3 +40,14 @@
 		@content;
 	}
 }
+
+/* 
+ * EXCEPTIONS
+ * Please be careful with using these classes. They don't follow the mobile-first approach but might be extremely useful in some use-cases.
+ */
+
+@mixin extremely-needed-mobile {
```

Comment:
Trying to think what other name would make sense for these exception cases. What do you think about using a `exception` prefix?

Reply:
How about `discouraged-mobile`? Open to other ideas too. I just want folks to know they need to thing twice before they use this mixin (and that they can't easily find it unless they look in here). I think we could also use a comment explaining why it's named the way it is.

Reply:
Not needed? Could we use mobile-first css for table-responsive?

Reply:
@jdanyow that was the going opinion for quite some time; we've come up with a few things were it's just a lot easier to target mobile. Not something we want to do generally, but there are definitely a few places were doing mobile first is a big pain. We (I) thought we could name it something that no one would guess and that might do a double take before using it.

Reply:
An example I came across was a full width button on mobile. Pretty annoying to try and unset widths on a higher resolution.

```
.button.button-full-width-mobile {
  width: 100%;
  @tablet {
    ??? 
  }
}

Reply:
Renamed to `discouraged-mobile`